data-serving/data-service/test/controllers/case.test.ts

```diff
@@ -23,8 +23,11 @@ const invalidRequest = {
     demographics: { ageRange: { start: 400 } },
 };
 
+const realDate = Date.now;
+
 beforeAll(async () => {
     mongoServer = new MongoMemoryServer();
+    global.Date.now = jest.fn(() => new Date('2020-12-12T12:12:37Z').getTime());
```

Comment:
ah, interesting. I've used time-warp libraries to mock the date in tests before, but this is much simpler.